tools/integration_tests/tokenserver/pytest.ini

```
@@ -0,0 +1,7 @@
+[pytest]
+markers =
+    local_integration: marks tests for running as local integration tests
+    live_integration: marks tests for running as live integration tests
+
+adopts =
```

I noticed this due to a warning in the log: `/usr/local/lib/python3.9/dist-packages/_pytest/config/__init__.py:1441: PytestConfigWarning: Unknown config option: adopts`
```suggestion
addopts =
```